### PR TITLE
Fizz Buzz in PHP

### PR DESCRIPTION
**Pull request by jrg94**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

File: archive/p/php/fizz-buzz.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    $output .= 'Buzz';
+  }
+
+  if (!$output)
+  {
+    $output = $i;
+  }
+
+  echo $output . "\n";
```
Comment (chrboe) [inline archive/p/php/fizz-buzz.php]: We should be using a consistent quote style.

Also, I know the interpreter doesn't complain, but it's considered good practice (I think) to have that `? >` at the end of the file

Comment (jrg94) [reply archive/p/php/fizz-buzz.php]: Both of these have their reasons. I'll work backwards through your comments:

In PHP, it's considered good practice to omit the closing PHP tag if the file only contains PHP code. (Source: [PSR-2](https://www.php-fig.org/psr/psr-2/#22-files))

As for strings, we probably could make them all the same. However, the quotes in PHP actually have a semantic difference. Single quotes treat strings literally--meaning *most* escape sequences will not be interpreted. Double quotes treat escape sequences as expected. (Source: [PHP Manual](http://php.net/manual/en/language.types.string.php))

If we still want consistent quotes, we should make them all double quotes.

Comment (chrboe) [reply archive/p/php/fizz-buzz.php]: > In PHP, it's considered good practice to omit the closing PHP tag if the file only contains PHP code. (Source: PSR-2)

Fair enough. Seems like I haven't written PHP in too long :slightly_smiling_face: 

About the quotes: it's ultimately up to you, I just wanted to point out the discrepancy there.

Comment (jrg94) [reply archive/p/php/fizz-buzz.php]: I'll swap the quotes over to double quotes, and we can call it even. 👍

**Pull request merged**